recyclerviewfastscroller/src/main/java/xyz/danoz/recyclerviewfastscroller/calculation/progress/VerticalLinearLayoutManagerScrollProgressCalculator.java

```diff
@@ -26,6 +26,8 @@ public float calculateScrollProgress(RecyclerView recyclerView) {
         int lastFullyVisiblePosition = layoutManager.findLastCompletelyVisibleItemPosition();
 
         View visibleChild = recyclerView.getChildAt(0);
+        if (visibleChild == null)
+            return 0;
```

Comment:
Why would you add the extra brackets? It'd work without them, but you can still add them yourself. No need for extra PR imho..